content_script.js

```diff
@@ -192,19 +192,17 @@ class HTMLSerializer {
   }
 
   /**
-   * Get the a URL object for the value of the elements src attribute.
+   * Get a URL object for the value of the elements src attribute.
    *
-   * @param {Element} element The element for which to retrive the URL.
+   * @param {Element} element The element for which to retrieve the URL.
    * @return {URL} The URL object.
    */
-   // TODO(sfine): Ensure that this is robust.
   srcURL(element) {
```

Comment:
Why do we need to create an img with the url before constructing URL, vs. just constructing off of element.attributes.src.value directly? It looks like this function has a side effect re: creating an img in the document and perhaps loading the content, worth elaborating in function documentation?